helm/exim/values.yaml

```diff
@@ -74,16 +83,38 @@ tolerations: []
 
 affinity: {}
 
+aadIdentityName: mailrelay
+usePodIdentity: "false"
+managedIdentityClientId: c4acf9cf-3120-4b87-baa9-3916ffd293eb
+
+authKeyVaultName: sds-mailrelay
+authKeyVaultSecrets: []
+tenantId: 531ff96d-0ae9-462a-8d2d-bec7c0b42082
+
```

Comment:
parameter values. certificate name/vault would be made more dynamic in next iterations